chalice/app.py

```diff
@@ -678,6 +678,7 @@ def _get_view_function_response(self, view_function, function_args):
                 body = stack_trace
                 headers['Content-Type'] = 'text/plain'
             else:
+                self.log.error("%s", traceback.format_exc())
```

Comment:
is it easier to use self.log.exception()

Reply:
I would agree, looking at this again I think should be logging at the `EXCEPTION` level. The reason that I chose to log this at the `ERROR` level is because the issue specified to log at the `ERROR` level.